docs/production/deployment.md

```diff
@@ -229,6 +229,10 @@ To protect against [SSRF][ssrf], Zulip 4.8 and above default to
 routing all outgoing HTTP and HTTPS traffic through
 [Smokescreen][smokescreen], an HTTP `CONNECT` proxy; this includes
 outgoing webhooks, website previews, and mobile push notifications.
+The Camo image proxy does not default to routing its traffic through
+Smokescreen if it is local, as Camo includes logic to deny access to
+private subnets; it will use a custom proxy if configured, and this
```

Comment:
Maybe do a `.` rather than a `;`.  And maybe replace "it will use" with something like: "If you configure a custom outgoing proxy for your Zulip server, ."

Maybe "By default, camo will be automatically configured to use a custom outgoing proxy, but does not use smokescreen by default because Camo includes similar logic to deny access to private subnets. You can [override]() this default camo configuration if desired."